### PR TITLE
Es mp some ui work dsdeepb 843

### PR DESCRIPTION
**Pull request by eddiebroad**

### CODE REVIEW

**Comment by MattPutnam** [comment]
I'm in favor of merging this.  It unblocks the other stories.  Objections?

**Comment by dmohs** [comment]
I'm fine with merging if you want to @MattPutnam. I may add some comments to this PR anyway just to track refactoring work.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs [hunk 1]
```diff
@@ -6,6 +6,7 @@
     [org.broadinstitute.firecloud-ui.page.workspace.workspace-summary :refer [render-workspace-summary]]
     [org.broadinstitute.firecloud-ui.page.workspace.workspace-data :refer [render-workspace-data]]
     [org.broadinstitute.firecloud-ui.page.workspace.method-configs :refer [render-method-configs]]
+    [org.broadinstitute.firecloud-ui.page.workspace.submissions :refer [render-submissions]]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs]: Update to follow new pattern.

**Comment by MattPutnam** [comment]
@eddiebroad This needs a rebase now

**Comment by eddiebroad** [comment]
I modified some "workspace" (name and namespace) getting after a rebase operation and made sure the URL and the mock data had the proper text in it (as opposed to nil).

**Comment by eddiebroad** [comment]
Not sure why a test is failing?  I got compilation locally...
wm8b1-75c:firecloud-ui esalinas$ git branch|grep "*" &&   git log|head && ./script/dev/build-once.sh 
* es_mp_some_UI_work_DSDEEPB-843
commit e53be910c6b92735e46a461e0e85d0cac04940c2
Author: Edward A. Salinas <esalinas@broadinstitute.org>
Date:   Fri Aug 21 16:10:39 2015 -0400

    extract workspace according to alternative method
    
    update workspace getter calls
    
    addition of submitter in render-single-submissin
    modification of whitespace for fitting < 100 chr per line
Compiling ClojureScript.
Compiling "target/compiled.js" from ["src/cljs"]...
Successfully compiled "target/compiled.js" in 4.422 seconds.
wm8b1-75c:firecloud-ui esalinas$

**Comment by dmohs** [comment]
Let's get this in.

**Pull request merged**